app/controllers/api/publishers_controller.rb

```diff
@@ -1,8 +1,4 @@
 class Api::PublishersController < Api::BaseController
-  before_action :require_verified_publisher,
-    only: %i(notify update_legal_form)
```

Comment:
should this filter still apply to `#notify`?

Reply:
Sorry, this one slipped by.

Reply:
@ayumi ok, fixed